src/esploader.ts

```diff
@@ -95,38 +95,51 @@ export class ESPLoader {
     private baudrate: number,
     private terminal: IEspLoaderTerminal,
     private rom_baudrate = 115200,
+    private debugLogging = false,
   ) {
     this.IS_STUB = false;
     this.chip = null;
     this.FLASH_WRITE_SIZE = 0x4000;
     if (this.terminal) {
       this.terminal.clean();
     }
+    this.debugLogging = false;
```

Comment:
This will always override the passed in value. Because you already have a visibility modifier in the constructor args (`private debugLogging`), TypeScript will automatically assign it as an instance field.

Reply:
Thanks for noticing this! It was a leftover from me testing something. I will remove this line.